src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -116,6 +109,19 @@ def _update_load_manager_args(args: argparse.Namespace) -> argparse.Namespace:
     return args
 
 
+def _infer_prompt_source(args: argparse.Namespace) -> argparse.Namespace:
+    if args.input_dataset is not None:
+        args.prompt_source = PromptSource.DATASET
+        logger.info(f"Input source is the following dataset: {args.input_dataset}")
+    elif args.input_file is not None:
+        args.prompt_source = PromptSource.FILE
+        logger.info(f"Input source is from the following file: {args.input_file}")
+    else:
+        args.prompt_source = PromptSource.SYNTHETIC
+        logger.info("Input source is from synthetic data")
```

Comment:
Do we want to make this only log in verbose mode? (e.g. logger.verbose; if we have not yet set this up, we should for optional logs like this one)

Reply:
Tagged @nv-hwoo to weigh in on what to do here (in a separate comment, since this one got pushed out of the code review window). I would rather not use new behavior as a reason for logging, because then users will get used to it being logged or we'll forget, and it will stay there. If it has to do with the minimal information the user needs to know about the tool starting or running, then we would want to log to info. Otherwise, we should start using verbose logging IMO. We don't need to start doing that in this PR though if both of you disagree with this approach.

Reply:
Elias moved these to debug mode as fast solution until we have more time to add verbose logging.